gensim/utils.py

```diff
@@ -1154,29 +1154,29 @@ def keep_vocab_item(word, count, min_count, trim_rule=None):
             return default_res
 
 
-def check_output(stdout=subprocess.PIPE, *popenargs, **kwargs):
-    """
-    Run command with arguments and return its output as a byte string.
-    Backported from Python 2.7 as it's implemented as pure python on stdlib.
-
-    >>> check_output(args=['/usr/bin/python', '--version'])
-    Python 2.6.2
-    Added extra KeyboardInterrupt handling
+def check_output(args, flag=True):
```

Comment:
My branch is upto date with develop. I ran ```python setup.py test``` with the appropriate env variable. Two tests failed but these are tests unrelated to what I am working on. 

What should I do now?